src/lib/broadcast.jl

```diff
@@ -251,43 +251,44 @@ end
   return y, bc_fwd_back
 end
 
-@init @require CUDA="052768ef-5323-5732-b1bb-66c8b64840ba" begin
+@init @require GPUArrays = "0c68f7d7-f131-5f86-a1c3-88cf8149b2d7" begin
 
-  const CuArrayStyle = CUDA.AbstractGPUArrayStyle
+  # const CuArrayStyle = AbstractGPUArrayStyle
 
-  if isdefined(CUDA, :cufunc)  # CUDA < 3.0
+  # if isdefined(CUDA, :cufunc)  # CUDA < 3.0
 
-    @eval @adjoint broadcasted(::CuArrayStyle, f, args...) =
-      broadcast_forward(CUDA.cufunc(f), args...)
+  #   @eval @adjoint broadcasted(::CuArrayStyle, f, args...) =
+  #     broadcast_forward(CUDA.cufunc(f), args...)
 
-  else # CUDA >= 3.0 -- don't need cufunc(f). 
-       # Ordinary broadcasting calls broadcast_forward anyway when certain its' safe,
-       # so perhaps this can be deleted? Possible edge case here:
-       # https://github.com/FluxML/Zygote.jl/pull/1018#issuecomment-873629415
+  # else # CUDA >= 3.0 -- don't need cufunc(f). 
+  #      # Ordinary broadcasting calls broadcast_forward anyway when certain its' safe,
+  #      # so perhaps this can be deleted? Possible edge case here:
+  #      # https://github.com/FluxML/Zygote.jl/pull/1018#issuecomment-873629415
 
-    @eval @adjoint broadcasted(::CuArrayStyle, f, args...) =
-      broadcast_forward(f, args...)
+  #   @eval @adjoint broadcasted(::CuArrayStyle, f, args...) =
+  #     broadcast_forward(f, args...)
 
-  end
+  # end
 
-  @adjoint (::Type{T})(xs::Array) where {T <: CUDA.CuArray} =
```

Comment:
Good opportunity to change these over to `rrule(::ZygoteRuleConfig, ...)` instead? IME they're much easier to debug than `@adjoint` definitions.